### ISSUE TITLE
action_not_found Webhook connector

### ISSUE THREAD

**Issue by lizhongren031** [emqx/emqx#13979]
### What happened?

服务器没做任何修改和设置使用一段时间，就报找不到相应的连接器，后续的钩子事件都无法收到。在webhook页面点测试是正常没问题，最后只能重启mq服务就正动恢复正常
2024-10-11T10:43:08.569954+08:00 [error] msg: action_not_found, connector: <<"connector:http:VMS_WH_D">>, action_id: <<"action:http:VMS_WH_D:connector:http:VMS_WH_D">>
2024-10-11T10:43:08.570358+08:00 [error] msg: action_not_found, connector: <<"connector:http:VMS_WH_D">>, action_id: <<"action:http:VMS_WH_D:connector:http:VMS_WH_D">>
2024-10-11T10:43:08.957817+08:00 [error] msg: action_not_found, connector: <<"connector:http:VMS_WH_D">>, action_id: <<"action:http:VMS_WH_D:connector:http:VMS_WH_D">>
2024-10-11T10:43:08.958304+08:00 [error] msg: action_not_found, connector: <<"connector:http:VMS_WH_D">>, action_id: <<"action:http:VMS_WH_D:connector:http:VMS_WH_D">>
2024-10-11T10:43:09.064853+08:00 [error] msg: action_not_found, connector: <<"connector:http:VMS_WH_D">>, action_id: <<"action:http:VMS_WH_D:connector:http:VMS_WH_D">>
2024-10-11T10:43:10.057179+08:00 [error] msg: action_not_found, connector: <<"connector:http:VMS_WH_D">>, action_id: <<"action:http:VMS_WH_D:connector:http:VMS_WH_D">>
2024-10-11T10:43:10.057667+08:00 [error] msg: action_not_found, connector: <<"connector:http:VMS_WH_D">>, action_id: <<"action:http:VMS_WH_D:connector:http:VMS_WH_D">>
2024-10-11T10:43:10.057877+08:00 [error] msg: action_not_found, connector: <<"connector:http:VMS_WH_D">>, action_id: <<"action:http:VMS_WH_D:connector:http:VMS_WH_D">>
2024-10-11T10:43:11.552788+08:00 [error] msg: action_not_found, connector: <<"connector:http:VMS_WH_D">>, action_id: <<"action:http:VMS_WH_D:connector:http:VMS_WH_D">>

### What did you expect to happen?

没做过任何变动，正常应该没问题才对

### EMQX version

<details>
docker emqx/emqx:5.8.0 
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
CentOS Linux release 7.9.2009 (Core)
</details>


### Log files

<details>

</details>

**Comment by qzhuyan** [comment]
Is it a cluster? Could you provide logs?

**Comment by xmfplus** [comment]
5.8.2 Appears again:
2024-11-21T06:32:59.095184+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:32:59.569998+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:00.711893+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:04.243016+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:04.739009+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:06.842813+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:07.383091+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:07.717757+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:08.661518+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:09.133273+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:09.388679+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:09.388876+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:10.022354+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>
2024-11-21T06:33:12.484317+00:00 [error] msg: action_not_found, connector: <<"connector:http:deyecloud">>, action_id: <<"action:http:deye_webhook_publish:connector:http:deyecloud">>

**Comment by xmfplus** [comment]
v5.8.2 has not been fixed yet and the same error occurs again

**Comment by id** [comment]
We are looking into this.

**Comment by thalesmg** [comment]
Should have been fixed by https://github.com/emqx/emqx/pull/14319, which will be released in 5.8.4.
